backend/foodbank_southlondon/api/events/models.py

```diff
@@ -6,6 +6,11 @@
 from foodbank_southlondon.api import models, rest
 from foodbank_southlondon.api.requests import models as requests_models
 
+class ResponseType(enum.Enum):
```

Comment:
If I don't open the new tab when the user clicks submit the browser will likely block it as a pop-up. Rather than get people to "Allow Pop Ups" for the app I use a workaround where I open a new tab when they click, then redirect it to the right URL once generated. In order to do this, I need to know what the response type is going to be for an action so I've generalised the `returns_pdf` into an enum for this.

Reply:
Good shout, good implementation. Thanks for this.